Entities/Common/Attacks/PlantHitEffects.as

```diff
@@ -5,25 +5,16 @@ f32 onHit(CBlob@ this, Vec2f worldPoint, Vec2f velocity, f32 damage, CBlob@ hitt
 {
 	if (damage > 0.05f) //sound for all damage
 	{
-		//read customdata for hitter
-		switch (customData)
+		if (customData != 7 && customData != 8) // not fire or burn
```

Comment:
not a big deal but since this is base kag it is probably better to use the actual enums from hitters.as just for reading purposes. you can disregard this comment though if wanted

Reply:
I changed it.